front/src/components/boxs/device-in-room/device-features/sensor-value/BinaryDeviceValue.jsx

```diff
@@ -10,10 +10,24 @@ const DANGER_ON_VALUE_SENSORS = [
   DEVICE_FEATURE_CATEGORIES.BATTERY_LOW,
   DEVICE_FEATURE_CATEGORIES.TAMPER
 ];
+const DANGER_ON_VALUE_SENSORS_TYPES = [
```

Comment:
I think we should validate based on the “Category + Type” combination rather than just the type alone. In many cases, the type by itself isn’t enough to accurately determine a feature. Let's do it right for future developments :)